### ISSUE TITLE
Date format assumption in documentation

### ISSUE THREAD

**Issue by boutros55** [MicrosoftDocs/OfficeDocs-Support#848]
The text of the document explaining how dates with two digits for the year when entered are interpreted is correct for people who use the USA format of month/day/year but incorrect for people in other parts of the world that use day/month/year format. My date of birth is 10/06/1955, that is 10th of June 1955, not October 6th 1955.
---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a47247d9-b661-ab7a-a20a-5e074cfee624
* Version Independent ID: 6f5e07f7-bcd1-9253-9818-13de415d21e6
* Content: [Excel works with two-digit year numbers - Office](https://docs.microsoft.com/en-us/office/troubleshoot/excel/two-digit-year-numbers)
* Content Source: [Office/Client/excel/two-digit-year-numbers.md](https://github.com/MicrosoftDocs/OfficeDocs-Support/blob/public/Office/Client/excel/two-digit-year-numbers.md)
* Product: **office-perpetual-itpro**
* GitHub Login: @simonxjx
* Microsoft Alias: **v-six**

**Comment by RAJU2529** [comment]
@boutros55 . i understood your point, I am living in India, so the india date format as day/month/year.
i  checked in excel sheet, I typed  16th October 2020,   data as formated as 16/10/2020 and  16-10-2020, this only possible when i change the regional format as  English (India)  other than English (UnitedStates) in windows settings under date &time.

If we choose  English (UnitedStates), the typed entry has been changed to  10/16/2020. 
The date format in EXCEL SHEET can be modified or  customized only in windows settings
